tox.ini

```diff
@@ -19,8 +20,9 @@ setenv =
     t44:  _DEP_PYTEST=pytest==4.4.2
     t45:  _DEP_PYTEST=pytest==4.5.0
 
-    {t310,t40,t41,t43}: _DEP_PYTESTXDIST=pytest-xdist==1.27.0
-    {t44,t45}: _DEP_PYTESTXDIST=pytest-xdist==1.28.0
+    x22: _DEP_PYTESTXDIST=pytest-xdist==1.22.0
+    x27: _DEP_PYTESTXDIST=pytest-xdist==1.27.0
+    x28: _DEP_PYTESTXDIST=pytest-xdist==1.28.0
```

Comment:
That looks nice/better in general.
How about using more explicit names here, i.e. "xdist122" etc?  (or "xdist1.22" if that is supported by tox)

Reply:
I was continuing with the style already here. Do you want me to expand "t" to "pytest" too?

Reply:
Yeah, I know.
I prefer longer names - and would also welcome it with other factors.

@ionelmc ?